components/StatusDropdown/styles.ts

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const FilterDropdownStyles = styled.div`
+  position: relative;
+  background: ${COLORS.white};
+  width: 10.5rem;
+  border-radius: 0.5rem;
+  padding-bottom: 0.5rem;
+`;
+
+export const OptionTitleStyles = styled.p<{ color: string }>`
+  background: ${({ color }) => `${color}10`};
+  border-radius: 0.3rem;
+  padding: 0.1rem 0.4rem;
+  display: inline-block;
+`;
+
+export const CheckboxStyles = styled.input`
+  cursor: pointer;
```

Comment:
add `margin-left: 0rem;` for some reason the button is not directly aligned with the apply button which we want